openshift/scripts/oc_deploy_nats.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh -l
```

Comment:
Would we deploy this every time we deploy? Is there going to be one per PR? Or do we just have one big nats server?

Reply:
For the spike I've just added this script to run locally -- it's not hooked up to PR or prod deployments

Reply:
Updated README at `openshift/README.md`